src/components/pages/landing/workshops/WorkshopsList/WorkshopsList.module.css

```diff
@@ -1,63 +1,133 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
 
-.workshopsList{
+.all{
```

Comment:
Mało deskryptywna nazwa klasy, lepszą nazwą będzie na przykład workshopsListContainer, container jest fajnym słowem bo jest to element, od którego zaczyna się cała struktura.